pandas/core/tools/timedeltas.py

```diff
@@ -68,8 +68,14 @@ def to_timedelta(arg, unit=None, errors="raise"):
 
     Returns
     -------
-    timedelta64 or numpy.array of timedelta64
-        Output type returned if parsing succeeded.
+    timedelta
+        If parsing succeeded.
+        Return type depends on input:
+
+        - list-like: TimedeltaIndex of timedelta64 dtype
+        - Series: Series of timedelta64 dtype
+        - Timedelta: Timedelta
```

Comment:
Nit: Could you summarize this and line below as `- scalar: Timedelta`. scalar includes `str`, `datetime.timedelta`, `np.timedelta64`, etc.